cmd/plugin/node_server_test.go

```diff
@@ -273,6 +276,165 @@ func TestNodePublishVolumeSync(t *testing.T) {
 	assert.ErrorContains(t, err, "not found")
 }
 
+// Check test/integration/node-server/README.md for how to run this test correctly
+func TestMetrics(t *testing.T) {
```

Comment:
Test result looks like this:
<img width="1593" alt="image" src="https://github.com/warm-metal/csi-driver-image/assets/34534103/5bb71151-9836-4c3b-92b1-9a3203b80c71">